ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -3355,6 +3355,7 @@ peer-filter PF2
    30 match as-range 65000 result accept
 !
 router bfd
+   slow-timer 5000
    interval 900 min-rx 900 multiplier 50 default
    multihop interval 300 min-rx 300 multiplier 3
    local-address 192.168.255.1
```

Comment:
According to EOS CLI `slow-timer` will come after `multihop interval`

```
   interval 900 min-rx 900 multiplier 50 default
   multihop interval 300 min-rx 300 multiplier 3
   slow-timer 5000
   local-address 192.168.255.1
```

You would have to change the order in router-bfd.j2

Reply:
@Vibhu-gslab which EOS version are you testing the order on?

I used 4.26.3 and the order there is:

```
router bfd
   slow-timer 5000
   local-address 1.1.1.1
   session stats snapshot interval dangerous 1
   interval 300 min-rx 300 multiplier 10 default
   multihop interval 50 min-rx 50 multiplier 3
   !
   sbfd
      initiator interval 50 multiplier 5
!
```

Reply:
thanks, let me update this and also update my testswitch to 4.32